extensions_admin/pulp_rpm/extensions/admin/export.py

```diff
@@ -52,6 +53,8 @@
 OPTION_ISO_SIZE = PulpCliOption('--iso-size', DESC_ISO_SIZE, required=False,
                                 parse_func=parsers.parse_optional_positive_int)
 
+FLAG_MANIFEST = PulpCliFlag('--' + constants.CREATE_PULP_MANIFEST, DESC_MANIFEST, ['-m'])
```

Comment:
This one is named ```FLAG_``` and the others are named ```OPTION_```. No preference on my part, but it might be nice if they were consistent.